lib/process-services-cloud/src/lib/process/start-process/components/start-process-cloud.component.ts

```diff
@@ -115,39 +116,65 @@ export class StartProcessCloudComponent implements OnChanges, OnInit, OnDestroy
     processDefinitionSelection: EventEmitter<ProcessDefinitionCloud> = new EventEmitter<ProcessDefinitionCloud>();
 
     processDefinitionList: ProcessDefinitionCloud[] = [];
-    processDefinitionCurrent: ProcessDefinitionCloud;
+    processDefinitionCurrent?: ProcessDefinitionCloud;
     errorMessageId: string = '';
-    processForm: UntypedFormGroup;
     processPayloadCloud = new ProcessPayloadCloud();
     filteredProcesses: ProcessDefinitionCloud[] = [];
-    isLoading = false;
-    isFormCloudLoaded = false;
-    formCloud: FormModel;
     staticMappings: TaskVariableCloud[] = [];
-    resolvedValues: TaskVariableCloud[];
+    resolvedValues?: TaskVariableCloud[];
 
     protected onDestroy$ = new Subject<boolean>();
+
+    isProcessStarting = false;
+    isFormCloudLoaded = false;
+    isFormCloudLoading = false;
     processDefinitionLoaded = false;
-    loading$ = new BehaviorSubject<boolean>(!this.processDefinitionLoaded);
 
-    constructor(
-        private startProcessCloudService: StartProcessCloudService,
-        private formBuilder: UntypedFormBuilder,
-        private processNameCloudPipe: ProcessNameCloudPipe
-    ) {}
+    formCloud?: FormModel;
+    processForm = new FormGroup({
+        processInstanceName: new FormControl('', [
+            Validators.required,
+            Validators.maxLength(this.getMaxNameLength()),
+            Validators.pattern('^[^\\s]+(\\s+[^\\s]+)*$')
+        ]),
+        processDefinition: new FormControl('', [Validators.required, this.processDefinitionNameValidator()])
```

Comment:
did u pass empty by purpose instead of this.processDefinitionName ?

Reply:
we could remove 
```
this.processDefinition.setValue(this.processDefinitionName);

Reply:
The form is now initiated on the class level and `processDefinitionName` comes from an Input. So at the moment of the initialization, the `processDefinitionName` doesn't have a value yet